utils/sender/src/main.rs

```diff
@@ -146,6 +191,9 @@ fn main() -> Result<(), Box<dyn Error>> {
 		OnlineClient::from_backend(backend).await.unwrap()
 	}
 
+	let health_node_url = args.node_url.clone();
+	spawn_healthcheck_server(args.health_port, move || create_api(health_node_url.clone()));
```

Comment:
This will pass in a different API instance than the one used in the sender. Please check that we createa a new one at line 257.